SimakovaKY/lab work 1.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream> 
+#include <conio.h>
+using namespace std;
+char ch = _getch();
+int main() {
+	do {
+		
+		float  x;
+		float  y;
+		char sign;
+
+		cin >> x >> sign >> y;
+		switch (sign) {
+		case '+':
+			cout << "Result = " << x + y;
```

Comment:
Please extract `cout << "Result = " << x ? y;` out of switch..case in order to reduce code duplication here and below